docs/UserGuide.md

```diff
@@ -220,7 +220,7 @@ In addition, there are some _optional_ extra parameters you can use to customize
 * **`-since START_DATE`**: The start date of analysis. Format: `DD/MM/YYYY`<br>
   Example:`-since 21/10/2017`
 * **`-until END_DATE`**: The end date of analysis. The analysis excludes the end date. Format: `DD/MM/YYYY`<br>
-  Example:`-since 21/10/2017`
+  Example:`-until 01/10/2017`
```

Comment:
Keep the date as `21/10/2017`, as it is intended to prevent confusion on whether the date is `DD/MM` or `MM/DD` format